examples/fxa-geodb-example.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+  'use strict';
+  var geoDb = require('../src/fxa-geodb')({
```

Comment:
Not necessary. Node `require()` has special handling - https://nodejs.org/dist/v0.10.46/docs/api/modules.html#modules_all_together